examples/asset_create_example.js

```diff
@@ -0,0 +1,65 @@
+// Example: creating an asset
+
+const algosdk = require('../');
+const utils = require('./utils');
+
+const { SENDER } = utils.retrieveBaseConfig();
+
+async function main () {
+  const sender = algosdk.mnemonicToSecretKey(SENDER.mnemonic);
+
+  // generate accounts
+  const { addr: freezeAddr } = algosdk.generateAccount(); // account that can freeze other accounts for this asset
+  const { addr: managerAddr } = algosdk.generateAccount(); // account able to update asset configuration
+  const { addr: clawbackAddr } = algosdk.generateAccount(); // account allowed to take this asset from any other account
+  const { addr: reserveAddr } = algosdk.generateAccount(); // account that holds reserves for this asset
+
+  const feePerByte = 10;
+  const firstValidRound = 1000;
+  const lastValidRound = 2000;
+  const genesisHash = 'SGO1GKSzyE7IEPItTxCByw9x8FmnrCDexi9/cOUJOiI=';
+
+  const total = 100; // how many of this asset there will be
+  const assetName = "assetname";
+  const unitName = "unitname";
+  const url = "website";
+  const metadata = "fACPO4nRgO55j1ndAK3W6Sgc4APkcyFh"; // should be a 32-byte hash
```

Comment:
```suggestion
  const metadata = new Uint8Array(Buffer.from("664143504f346e52674f35356a316e64414b3357365367633441506b63794668", "hex")); // should be a 32-byte hash
```
  
Something like this would be a better example, since we really want to encourage passing in a Uint8Array instead of a string for this parameter.
 
Also, can you include `decimals` as a parameter in this section too?